src/rules/slack.js

```diff
@@ -13,6 +13,8 @@ function slackNotificationOnUserSignup(user, context, callback) {
   if (context.stats.loginsCount > 1 || context.protocol === 'oauth2-refresh-token') {
     return callback(null, user, context);
   }
+  // check this isn't a checkSession call  
+  if (context.request.query.prompt !== 'none'){
```

Comment:
Can we switch this to a guard clause? Check for that prompt and callback if it's there?